evap/staff/views.py

```diff
@@ -401,6 +401,22 @@ def semester_create(request):
     return render(request, "staff_semester_form.html", dict(form=form))
 
 
+@require_POST
+@manager_required
+def semester_make_active(request):
+    semester_id = request.POST.get("semester_id")
+    semester = get_object_or_404(Semester, id=semester_id)
+
+    previous_active_semester = Semester.objects.filter(is_active=True).first()
+    previous_active_semester.is_active = None
+    previous_active_semester.save()
```

Comment:
`Semester.objects.update(is_active=None)`
is one line instead of three and it gives the advantage that this doesn't crash if no semester is currently active (which you will have when you set up a new instance of the software - I think we should allow users to at least make a semester active in this case)